src/main/kotlin/graphics/scenery/volumes/Volume.kt

```diff
@@ -102,6 +102,23 @@ open class Volume(val dataSource: VolumeDataSource, val options: VolumeViewerOpt
     /** Plane equations for slicing planes mapped to origin */
     var slicingPlaneEquations = mapOf<SlicingPlane, Vector4f>()
 
+    /** Modes how assigned slicing planes interact with the volume */
+    var slicingMode = SlicingMode.NONE
+
+    enum class SlicingMode(val id: Int){
+        // Volume is rendered as it is
+        NONE(0),
```

Comment:
Please make the Enum members non-caps, so e.g. just `None`, `Cropping` etc 👍

Reply:
done